src/3D/L3DMesh.cpp

```diff
@@ -140,32 +137,34 @@ void L3DMesh::Load(IStream& stream)
 	_flags = header.flags;
 
 	/*if (IsBoned())
```

Comment:
if we use `#if 0` for block comments, then the formatting will probably stay as code